src/ai/backend/manager/config/volume.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Optional
+
+from pydantic import BaseModel, Field
+
+
+class VFolderTypeConfig(BaseModel):
+    user: Optional[dict[str, Any]] = Field(
+        default=None,
+        description="""
+        User vFolder type configuration.
+        When present, enables user-owned virtual folders.
+        Standard folder type for individual users.
+        """,
+    )
+    group: Optional[dict[str, Any]] = Field(
+        default=None,
+        description="""
+        Group vFolder type configuration.
+        When present, enables group-owned virtual folders.
+        Used for sharing files within a group of users.
+        """,
+    )
+
+
+class VFolderProxyConfig(BaseModel):
+    client_api: str = Field(
+        description="""
+        Client-facing API endpoint URL of the volume proxy.
+        Used by clients to access virtual folder contents.
+        Should include protocol, host and port.
+        """,
+        examples=["http://localhost:6021", "https://proxy1.example.com:6021"],
+    )
+    manager_api: str = Field(
+        description="""
+        Manager-facing API endpoint URL of the volume proxy.
+        Used by manager to communicate with the volume proxy.
+        Should include protocol, host and port.
+        """,
+        examples=["http://localhost:6022", "https://proxy1.example.com:6022"],
+    )
+    secret: str = Field(
+        description="""
+        Secret key for authenticating with the volume proxy manager API.
+        Must match the secret configured on the volume proxy.
+        Should be kept secure and not exposed to clients.
+        """,
+        examples=["some-secret-key"],
+    )
+    ssl_verify: bool = Field(
+        default=True,
+        description="""
+        Whether to verify SSL certificates when connecting to the volume proxy.
+        Should be enabled in production for security.
+        Can be disabled for testing with self-signed certificates.
+        """,
+        examples=[True, False],
+    )
+    sftp_scaling_groups: Optional[list[str]] = Field(
+        default=None,
+        description="""
+        List of scaling groups that can access this volume via SFTP.
+        Controls which compute sessions can connect to this volume using SFTP.
+        If None, all scaling groups can access the volume.
```

Comment:
```suggestion
        List of SFTP scaling groups that the volume is mapped to.
        Controls which scaling groups can create SFTP sessions for this volume.
```
The description is incorrect